### ISSUE TITLE
Additional library: Chromium CSS selector generator

### ISSUE THREAD

**Issue by dandv** [fczbkk/css-selector-generator-benchmark#2]
Somehow I neglected to mention in my [list of alternative libraries](https://github.com/fczbkk/css-selector-generator/issues/2) Chromium's `WebInspector.DOMPresentationUtils.cssPath` [method](https://chromium.googlesource.com/chromium/blink/+/master/Source/devtools/front_end/components/DOMPresentationUtils.js) from the DevTools frontend. It was extracted into [a gist](https://gist.github.com/asfaltboy/8aea7435b888164e8563) (which is now a bit out of date) and an [npm package](https://www.npmjs.com/package/cssman) which [may or may not be on GitHub](https://github.com/maximilianschmitt/iniquest/issues/1).

**Comment by fczbkk** [comment]
Added test for Chromium's DOM Inspector selector generator, see https://github.com/fczbkk/css-selector-generator-benchmark#chromiums-dompresentationutils

**It produces lots of non-unique selectors.** I guess it's because its results are not intended to be used as an unique CSS selector. They are meant to be used as a display path to currently inspected element.

I have tried to adapt the original [DOMPresentationUtils.js](https://chromium.googlesource.com/chromium/blink/+/master/Source/devtools/front_end/components/DOMPresentationUtils.js) for use in browser, but it was too complicated. It uses heavily modified `DOMNode.prototype` object.
